src/components/footer/navigation-menu/navigation-menu.tsx

```diff
@@ -9,9 +12,25 @@ export const NavigationMenu = ({
   title,
   menuElements,
 }: NavigationMenuProps) => {
+  const [arrow, setArrow] = useState<string>("down");
+
+  const ArrowChange = (id) => {
```

Comment:
function should not be named with capital letter

Reply:
ale ale jak to? Oo